packages/cqrs/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 9.0.0
+
+- Bumped `http` dependency to `1.0.0`.
+- **Breaking:** Bump minimum Dart version to 3.0.
```

Comment:
I like linking PRs in changelogs, like this:

```
- Bumped `http` dependency to `1.0.0` (#105)
- **Breaking:** Bump minimum Dart version to 3.0 (#105)
```

But that's just my quirk, I guess.

Reply:
It is definitely a good idea, easier to track down the change